.github/COMMIT_CONVENTION.md

```diff
@@ -0,0 +1,51 @@
+## Git Commit Message Convention
+
+> This is adapted from [Angular's commit convention](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular).
+
+#### TL;DR:
+
+Messages must be matched by the following regex:
+
+<!-- prettier-ignore -->
+```js
+/^(revert: )?(feat|fix|docs|refactor|perf|test|ci|chore|release)(\(.+\))?: .{1,50}/
+```
+
+#### Examples
+
+Appears under "Features" header:
+
+```
+feat(@material-svelte/button): add a specific new feature
+```
+
+Appears under "Bug Fixes" header:
+
+```
+fix(@material-svelte/button): fix a specific issue
+```
+
+### Type
+
+If the prefix is `feat`, `fix` or `perf`, it will appear in the changelog. However, if there is any [BREAKING CHANGE](#footer), the commit will always appear in the changelog.
+
+Other prefixes are up to your discretion. Suggested prefixes are `docs`, `chore`, `refactor`, and `test` for non-changelog related tasks.
+
+### Scope
+
+The scope should be a specific workspace-package (i.e. `@material-svelte/button`) or one of the scopes in the [configuration](../.commitlintrc.js)
+
+### Subject
+
+The subject contains a succinct description of the change:
+
+- use the imperative, present tense: "change" not "changed" nor "changes"
```

Comment:
just a comment: let's see who breaks these first xD
I would argue if someone really doesn't know how to git and stuff (to change their messages), that I help them a little more than before. I hope I remember that I said this, when push comes to shove.

Reply:
I'll bet: I'm the first to break it xD